environments/common.yaml

```diff
@@ -164,7 +164,25 @@ jobs:
       // Filter by name (with regular expression)
       headRegexFilter {
         // Regular expression
-        regex('main|PR-.*')
+        regex('main')
+      }
+    releaseMasterHeadRegexFilter: |-
+      // Filter by name (with regular expression)
+      headRegexFilter {
+        // Regular expression
+        regex('master')
+      }
+    releaseLtsHeadRegexFilter: |-
```

Comment:
Is it used? Or just in case we need it

Reply:
It is not used and here just in case we need it, I can remove it if you prefer?

Reply:
No leave it it's fine.